gef.py

```diff
@@ -10423,10 +10421,10 @@ def add_function_to_doc(self, function) -> None:
 
     def do_invoke(self, argv) -> None:
         self.dont_repeat()
-        gef_print(titlify("GEF - Convenience Functions"))
-        gef_print("These functions can be used as arguments to other "
-                  "commands to dynamically calculate values\n")
-        gef_print(self.__doc__)
+        gef_print(titlify("GEF - Convenience Functions"),
+                  "These functions can be used as arguments to other "
+                  "commands to dynamically calculate values\n",
```

Comment:
this instance is particularly confusing since we have both concatenated strings as well as multiple independent strings.

Reply:
I agree it's confusing, a comma is easiest to miss than a `\n` for me.
Maybe we should turn this into a discussion first about how to improve `gef_print`? When you mentioned it, I initially thought you wanted something more advanced (`rich` style) to make the formatting easier.

Reply:
Yeah, but also in this case we couldn't do the `\n` because the first 'arg' is wrapped in `titlify` so the auto concatenate won't work.

Reply:
True, even though using f-strings for `titlify` removes the problem I get your point.